fecgov/fec-cms
# [MERGE WITH GIT FLOW] 5507 admin fines calculator 2023

## Summary

- Resolves #5507 

Updating the administrative fines calculator for 2023

### Required reviewers

Internal teams have approved it and given us the go-ahead to publish to Production

## Impacted areas of the application

The [administrative fines calculator](https://www.fec.gov/legal-resources/enforcement/administrative-fines/calculating-administrative-fines/#calculating-a-fine) and its tests

## Screenshots

<img width="961" alt="image" src="https://user-images.githubusercontent.com/26720877/210644059-02f89eda-efba-4d4b-a43e-67ea7147b41e.png">

## Related PRs

None

## How to test
- Check Feature at /legal-resources/enforcement/administrative-fines/calculating-administrative-fines/#calculating-a-fine
- On the screen that offers dates, make sure the newest one is 2022-Present, like in the attached screenshot
